documentation/en/cli-lotus.md

```diff
@@ -2089,6 +2090,8 @@ COMMANDS:
      encode                            encode various types
      disputer                          interact with the window post disputer
      prune                             prune the stored chain state and perform garbage collection
+     create-evm-actor                  Create an new EVM actor via the init actor and return its address
+     invoke-evm-actor                  Invoke a contract entry point in an EVM actor
```

Comment:
Those command probably belong in the `evm` group. `chain` commands are mostly meant for interacting with the "low-level" chain (think block-headers), then `state` commands are for native filecoin state

Reply:
Moved to the `evm` command group.